contracts/TokenLocking.sol

```diff
@@ -39,10 +39,11 @@ contract TokenLocking is TokenLockingStorage, DSMath { // ignore-swc-123
     _;
   }
 
-  modifier tokenNotLocked(address _token) {
-    if (userLocks[_token][msg.sender].balance > 0) {
-      require(userLocks[_token][msg.sender].lockCount == totalLockCount[_token], "colony-token-locking-token-locked");
+  modifier tokenNotLocked(address _token, bool _force) {
+    if (_force) {
+      userLocks[_token][msg.sender].lockCount = totalLockCount[_token];
```

Comment:
I'm not wild about a modifier have an effect, but I guess it's fine. It's certainly very convenient to have this here.

Reply:
Yeah I say let's run with it but try not to do it too often.